flask/datahandle.py

```diff
@@ -270,26 +327,25 @@ def requestItem(date_in, loc_in, item_in, meal_in):
         for j in i['course']:
             for key, value in j.items():
                 if key == 'name':
-                    coursedata = j['menuitem']
-                    mealname = i['name']
-                    #Append matches to specified item to possiblematches list
-                    possiblematches = findMatches(coursedata, possiblematches, item_in, mealname)
+                    courseData = j['menuitem']
```

Comment:
Since you're changing variable names, the python preferred style for variables is snake_case ([pep8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)) all lowercase separated by underscores.  StudlyCaps are used for class names.

Sticking with the language conventions will make developing in the future easier! 

.NET and Java often use camelCase instead but there are project specific conventions.

Reply:
Do you want me to change all the variables in both files to be snake case? That might be better off in a new pull request just for formatting.

Reply:
Yeah, that sounds good, I can open a new issue about this and it can be done separate. Just noticed that the names were changing.